crates/forge_services/src/tool_services/fs_read.rs

```diff
@@ -74,8 +74,15 @@ impl<F: FileInfoInfra + EnvironmentInfra + InfraFsReadService> FsReadService for
             .await
             .with_context(|| format!("Failed to read file content from {}", path.display()))?;
 
+        let start_line = file_info.start_line as usize;
+        let lines: Vec<String> = content
+            .lines()
+            .enumerate()
+            .map(|(i, line)| format!("{}: {}", start_line + i, line))
```

Comment:
- update indentation
- Update tool docs and state that the tool returns files with line numbers by default.
- Add a flag for the agent control display with/without linenumbers.